### ISSUE TITLE
New account registrations not allowed on Discourse

### ISSUE THREAD

**Issue by tiarap00** [dask/community#398]
I've tried to register to Discourse with my GitHub account, but the following message prevents me to continue: New account registrations are not allowed at this time.

**Comment by hendrikmakait** [comment]
cc @jacobtomlinson, this seems to be related to the recent changes you made (https://dask.discourse.group/t/switch-to-github-only-for-new-user-registration/3164)

**Comment by jacobtomlinson** [comment]
That’s right, we had a lot of bot spam recently and switched it so that you need to log in via GitHub. As you opened this issue on GitHub you should be able to log straight in.

I’ll explore whether we can put a better message on the login page to avoid other folks running into this.

**Comment by hendrikmakait** [comment]
@jacobtomlinson: From what I understand, the issue described by @tiarap00 is that he's trying sign up via GitHub but can't. Using a newly-created dummy account, I've confirmed that new registrations via GitHub do not work and instead present `New account registrations are not allowed at this time.` as the error.

**Comment by jacobtomlinson** [comment]
Ah that's a shame! I tested it with a spare account but it used my email address with the plus sigh and some extra characters to make it unique. I guess Discourse just sees that as the same address and let me log in.

I've enabled registrations again for now. We will need to think of another solution to stop the bots.

**Comment by jacobtomlinson** [comment]
(Also I'm going to migrate this issue to dask/community as it's not related to dask/dask)

**Comment by tiarap00** [comment]
@hendrikmakait that's correct, this is the issue I was describing. I'm sorry I didn't categorize the issue properly.

**Comment by hendrikmakait** [comment]
> (Also I'm going to migrate this issue to dask/community as it's not related to dask/dask)

Thanks, I didn't have the permissions to do so.

**Comment by jacobtomlinson** [comment]
@guillaumeeb it looks like the bots are back after I enabled registration again. I'm going to disable it again for now. Are you able to take a deeper look into this?

**Comment by guillaumeeb** [comment]
@jacobtomlinson I don't think I've enough permissions on the Discourse forum to tweek authentication settings and search for a better solution...

**Comment by jacobtomlinson** [comment]
I'll open a discussion with the Dask steering group about bumping your permissions up.

**Comment by jacobtomlinson** [comment]
@guillaumeeb you now have admin rights on the forum :)
